README.md

```diff
@@ -33,7 +44,7 @@ LibreOffice-based online office suite with collaborative editing
 The settings of Collabora are at `https://office.domain.tld/loleaflet/dist/admin/admin.html`.
```

Comment:
The new URI for the admin is : https://office.domain.tld/browser/dist/admin/admin.html